packages/desktop/electron/lib/Ledger.js

```diff
@@ -197,4 +203,97 @@ class Ledger {
     }
 }
 
-export default new Ledger();
+class LedgerSpeculos extends Ledger {
+    async selectSeed(index, page, security) {
```

Comment:
Unnecessary, please remove!

Reply:
@laumair what exactly did you want to remove? whole `selectSeed` ?